engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -93,11 +91,13 @@ private void updateFullScale() {
      */
     public void update() {
         updateFullScale();
-        if (READ_ONLY_GBUFFER.getFbo().dimensions().areDifferentFrom(fullScale)) {
+        if (get(new ResourceUrn("engine:sceneOpaque")).dimensions().areDifferentFrom(fullScale)) {
```

Comment:
There's a lot of brackets and concatenations in there, making it somewhat hard to read. Breaking it into two lines will help. Of course it's all a matter of personal standards. I'm sure geniuses could write and easily read through the whole method in one line. But that's what PERL is for, really. :smile: